### PR TITLE
Add temperatures

### PR DESCRIPTION
**Pull request by rbudai98**

### CODE REVIEW

File: sdk/include/aditof/camera.h [hunk 1]
```diff
@@ -279,6 +279,20 @@ class SDK_API Camera {
      * @return Status
      */
     virtual Status adsd3500GetRadialThresholdMax(int &threshold) = 0;
+
+    /**
+     * @brief Get the sensor temperature threshold
```
Comment (dNechita) [inline sdk/include/aditof/camera.h:284]: I think the work "threshold" is not necessary.

File: sdk/include/aditof/camera.h [hunk 2]
```diff
@@ -279,6 +279,20 @@ class SDK_API Camera {
...
+     * @param[out] temperature
+     * @return Status
+     */
+    virtual Status adsd3500GetSensorTemperature(uint16_t &tmpValue) = 0;
+
+    /**
+     * @brief Get the laser temperature threshold
+     * @param[out] temperature
```
Comment (dNechita) [inline sdk/include/aditof/camera.h:292]: Do we know the measurement unit? If yes, can you mention it here in the description of the function?

File: sdk/src/connections/target/adsd3500_sensor.cpp [hunk 3]
```diff
@@ -815,6 +815,10 @@ aditof::Status Adsd3500Sensor::adsd3500_read_cmd(uint16_t cmd, uint16_t *data) {
 
     extCtrl.p_u8 = buf;
 
+    if (cmd == 0x0054 ||
+        cmd == 0x0055) //in case of temperature requests response time needed
+        usleep(90000);
```
Comment (dNechita) [inline sdk/src/connections/target/adsd3500_sensor.cpp:820]: How did you determine the 90 ms value?

File: sdk/include/aditof/camera.h (hunk 1 above)
Comment (rbudai98) [reply sdk/include/aditof/camera.h:284]: removed

File: sdk/include/aditof/camera.h (hunk 2 above)
Comment (rbudai98) [reply sdk/include/aditof/camera.h:292]: added measurement unit

File: sdk/src/connections/target/adsd3500_sensor.cpp (hunk 3 above)
Comment (rbudai98) [reply sdk/src/connections/target/adsd3500_sensor.cpp:820]: Witch consecutive trying, still waiting for confirm from the firmware side

Comment (rbudai98) [reply sdk/src/connections/target/adsd3500_sensor.cpp]: ADSD3500 need to process the last frame that sensor is currently working on in order to create a response. This line has been replaced with a generic solution based on frames per second of the sensor

**Review by dNechita — APPROVED** [review]

**Pull request merged**